lib/MusicBrainz/Server/Form/Role/ValidateUsername.pm

```diff
@@ -0,0 +1,22 @@
+package MusicBrainz::Server::Form::Role::ValidateUsername;
+use HTML::FormHandler::Moose::Role;
+use MusicBrainz::Server::Translation qw( l );
+
+sub validate_username
+{
+    my ($self, $field) = @_;
+
+    my $username = $field->value;
+    if (defined $username) {
+        if ($username =~ qr{^deleted editor \#\d+$}i) {
+            $field->add_error(l('This username is reserved for internal use'));
```

Comment:
I'd expect they are (this is the validation we were already doing for registering, I've just moved it to a role). Not sure about punctuation, https://github.com/metabrainz/musicbrainz-server/search?q=add_error&type=Code&utf8=%E2%9C%93 suggests the answer is "sometimes"